src/lib/structures/commands.ml

```diff
@@ -58,6 +60,8 @@ let print_aux fmt = function
       name print_goal_sort sort Expr.print e
   | ThAssume t ->
     Format.fprintf fmt "th assume %a" Expr.print_th_elt t
+  | Push -> Format.fprintf fmt "Push"
+  | Pop ->  Format.fprintf fmt "Pop"
 
 let print fmt decl = print_aux fmt decl.st_decl
```

Comment:
I probably never noticed that a function `print` exists in commands.ml. Has it been added recently ?

Reply:
You probably never noticed this function because it's in a comment :) maybe never used.